xtask/src/tests/mod.rs

```diff
@@ -83,6 +86,37 @@ impl TestArgs {
             }
         }
         let backend = &self.backend;
+        if self.wasm {
+            if matches!(backend, Backend::Sqlite) {
+                let mut command = Command::new("wasm-pack");
+                let out = command
+                    .args(["test", "--chrome", "--headless", "--features", "sqlite"])
+                    .current_dir(metadata.workspace_root.join("diesel"))
+                    .stderr(Stdio::inherit())
+                    .stdout(Stdio::inherit())
+                    .status()
+                    .unwrap();
+                if !out.success() {
+                    eprintln!("Failed to run wasm diesel unit tests");
+                    return false;
+                }
+                let mut command = Command::new("wasm-pack");
+                let out = command
+                    .args(["test", "--chrome", "--headless", "--features", "sqlite"])
+                    .current_dir(metadata.workspace_root.join("diesel_tests"))
+                    .stderr(Stdio::inherit())
+                    .stdout(Stdio::inherit())
+                    .status()
+                    .unwrap();
+                if !out.success() {
+                    eprintln!("Failed to run wasm integration tests");
+                    return false;
+                }
+                return true;
+            } else {
```

Comment:
Can we emit a helpful error message that says that only sqlite is supported for now?